src/cdk/layout/layout.md

```diff
@@ -2,18 +2,24 @@ The `layout` package provides utilities to build responsive UIs that react to sc
 
 ### BreakpointObserver
 
-`BreakpointObserver`  is a utility for evaluating media queries and reacting to changes in the results of those queries.
+A layout **breakpoint** is viewport size threshold at which a layout shift can occur. The viewport
+size ranges between breakpoints correspond to different standard screen sizes. 
+
+`BreakpointObserver` lets you evaluate media queries to determine the current screen size and
+react to changes when the viewport size changes and crosses a breakpoint.
+
+#### Check the current viewport size
+You can use the `isMatched` method to evaluate one or more media queries against the current
+viewport size.
 
-#### Evaluate against the current viewport
-The `isMatched` method is used to evaluate one or more media queries against the current viewport
-size.
 ```ts
 const isSmallScreen = breakpointObserver.isMatched('(max-width: 599px)');
 ```
 
 #### React to changes to the viewport
-The `observe` method is used to get an observable stream that will emit whenever one of the given
-media queries would have a different result.
+You can use the `observe` method to get an observable stream emits whenever the viewport size
```

Comment:
observable stream emits -> observable stream that emits

Reply:
Done